content/foundation/_index.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "The Matrix.org Foundation"
+sort_by = "weight"
+render = false
++++
+
+* [About](about) &ndash; The Matrix.org Foundation About page
```

Comment:
I think this is technically unreachable since its render=false and the about page itself takes this url.